spec/features/work_packages/details/inplace_editor/subject_editor_spec.rb

```diff
@@ -11,6 +11,7 @@
   let(:field) { WorkPackageField.new page, property_name }
 
   before do
+    maximize(page)
     allow(User).to receive(:current).and_return(user)
     visit project_work_packages_path(project)
```

Comment:
I am not too familiar with the shared contexts beforehand, but do you mean something like this?

```ruby 
shared_examples 'a field that requires a maximized window' do
  context do
    before do
       maximize(page)
    end
  end
end
```

with a call in the spec itself:

```ruby
# [...]

it_behaves_like 'a field that requires a maximized window'
```

?